content/docs/get-started/kubernetes/deploy-changes.md

```diff
@@ -60,7 +60,22 @@ Duration: 12s
 
 You can see we now have an `ip` [stack output]({{< relref "/docs/intro/concepts/stack#outputs" >}}) that we can `curl` to get the output from the service.
 
-> **Note:** minikube does not support type LoadBalancer; if you are deploying to minikube, make sure to run kubectl port-forward svc/frontend 8080:80 to forward the cluster port to the local machine and access the service via localhost:8080.
+<!-- markdownlint-disable blanks-around-fences -->
+> **Using Minikube:** Note that Minikube does not support type `LoadBalancer`. If you are deploying to Minikube, see the following example to run the `kubectl port-forward service/YOUR_SERVICE_NAME 8080:80` command to forward the cluster port to the local machine. Then, the service can be accessed via `curl http://localhost:8080`, which will get the same result as `curl $(pulumi stack output ip)` as in the environment without using Minikube.
+>
+> ```bash
+> $ kubectl get service
+> NAME             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
+> kubernetes       ClusterIP   10.96.0.1       <none>        443/TCP   26h
+> nginx-3hq3kux6   ClusterIP   10.96.185.206   <none>        80/TCP    15m
+> ```
+>
+> ```bash
+> $ kubectl port-forward service/nginx-3hq3kux6 8080:80
+> Forwarding from 127.0.0.1:8080 -> 80
+> Forwarding from [::1]:8080 -> 80
+> ```
```

Comment:
Still need to remove the `>` _inside_ the code blocks. After that, it should be ready to merge! Thanks, again for the improvements here!

~~~suggestion
$ kubectl get service
NAME             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
kubernetes       ClusterIP   10.96.0.1       <none>        443/TCP   26h
nginx-3hq3kux6   ClusterIP   10.96.185.206   <none>        80/TCP    15m
```
>
> ```bash
$ kubectl port-forward service/nginx-3hq3kux6 8080:80
Forwarding from 127.0.0.1:8080 -> 80
Forwarding from [::1]:8080 -> 80
```
~~~

Reply:
Hmm. My local Visual Studio Code does not properly render under Markdown preview without these block quote in the code block.

I will change according to your feedback. Pull request updated.

Reply:
I pulled down the changes in your PR and confirmed that Hugo renders it as expected with these changes. Thanks!.

Reply:
Great! Good to hear. I did not know that the website is using Hugo. 😅